lib/network/modules/components/shared/Label.js

```diff
@@ -699,7 +699,7 @@ class Label {
    * Explodes a piece of text into single-font blocks using a given markup
    * @param {string} text
    * @param {boolean|'md'|'markdown'|'html'} markupSystem
```

Comment:
While informative, is putting constants in the inline type tag legal? Again, not from your hand....

Reply:
Yes I believe it is. String literals are certainly legal in typescript.

Reply:
OK. And if the jsdoc parser doesn't choke on it either, I guess it's good.